packages/flutter_tools/lib/src/commands/build_apk.dart

```diff
@@ -385,7 +407,8 @@ Future<int> buildAndroid(
   String outputFile: _kDefaultOutputPath,
   String target,
   String flxPath,
-  ApkKeystoreInfo keystore
+  ApkKeystoreInfo keystore,
+  BuildVariant buildVariant: BuildVariant.develop
```

Comment:
You might want to make this a require parameter, or you could get strange behavior in the places where it's omitted by mistake?